x-pack/plugins/maps/public/embeddable/types.ts

```diff
@@ -35,9 +35,10 @@ interface MapEmbeddableState {
 }
 export type MapByValueInput = {
   attributes: MapSavedObjectAttributes;
-} & EmbeddableInput &
-  MapEmbeddableState;
-export type MapByReferenceInput = SavedObjectEmbeddableInput & MapEmbeddableState;
+} & EmbeddableInput & { filterByMapExtent?: boolean } & MapEmbeddableState;
```

Comment:
its not clear from this but i am guessing map center point and zoom level are part of the input ?

Reply:
center point and zoom and in `MapEmbeddableState.mapCenter`